charter.md

```diff
@@ -14,22 +14,34 @@ proxy. Moreover, it would not introduce potentially redundant or unnecessary rec
 an established request/response semantic that can set up and configure flows for different services.
 
 The primary goal of this working group is to develop mechanism(s) that allow configuring and concurrently running
-multiple proxied stream- and datagram-based flows inside an HTTPS connection. These mechanism(s) are collectively called
+multiple proxied stream- and datagram-based flows inside an HTTP connection. These mechanism(s) are collectively called
 MASQUE. The group will specify HTTP and/or HTTP/3 extensions to enable this functionality.
 
-The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
-Server-initiated services are out of scope. The working group will first deliver a protocol solution for UDP CONNECT and
-a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
-IP proxying.
+The group will focus on extensions for a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
+MASQUE is only chartered to work on services that the client requests from the proxy.
```

Comment:
```suggestion
Since the MASQUE working group already delivered protocol solutions for CONNECT-UDP and CONNECT-IP,
the group will now focus on extensions for these two client-initiated services.
```

Reply:
@martinduke please have another look. We would like to close the consensus call and hand you the token to move this forward ASAP!